### PR TITLE
Improve several hyprctl commands

### PR DESCRIPTION
**Pull request by riley-martin**
#### Describe your PR, what does it fix/add?
This adds better help for some hyprctl subcommands, particularly `dispatch`, `keyword`,
`hyprpaper` and `setcursor`. It also makes them return a nonzero exit status to the shell
when they do not have enough arguments.

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
No, not that I am aware of.

#### Is it ready for merging, or does it need work?
Works for me (tm), but yes, it should be ready to merge.

### CODE REVIEW

**Review by fufexan — APPROVED** [review]

**Comment by NotAShelf** [comment]
I'm all in for this PR, but why those commands in those particular?

**Comment by riley-martin** [comment]
Mostly because they were ones that took additional arguments, and gave somewhat unclear 
error messages when not enough arguments were provided. `getoption` also takes another argument
but since it is just one, its a little easier to understand IMO; also changing it would be more
complicated. If you think the others should be improved, though, I can work on it.

File: hyprctl/main.cpp [hunk 1]
```diff
@@ -172,28 +175,33 @@ void keywordRequest(int argc, char** argv) {
...
     request(rq);
+    return 0;
 }
 
-void hyprpaperRequest(int argc, char** argv) {
+int hyprpaperRequest(int argc, char** argv) {
     if (argc < 4) {
-        std::cout << "hyprpaper requires 2 params";
-        return;
+        std::cout << "Usage: hyprctl hyprpaper <commang> <arg>\n\
```
Comment (vaxerski) [inline hyprctl/main.cpp:183]: commang

File: hyprctl/main.cpp [hunk 2]
```diff
@@ -158,12 +159,14 @@ void dispatchRequest(int argc, char** argv) {
...
+    return 0;
 }
 
-void keywordRequest(int argc, char** argv) {
+int keywordRequest(int argc, char** argv) {
     if (argc < 4) {
-        std::cout << "keyword requires 2 params";
-        return;
+        std::cout << "Usage: hyprctl keyword <keyword> <args>\n\
+            Execute a hyprland keyword with the given arguments";
```
Comment (vaxerski) [inline hyprctl/main.cpp:168]: arguments here, argument above. Decide

**Comment by vaxerski** [comment]
can you plox squash

**Pull request merged**